include/uapi/sound/sof/tokens.h

```diff
@@ -121,4 +121,6 @@
 #define SOF_TKN_MUTE_LED_USE			1300
 #define SOF_TKN_MUTE_LED_DIRECTION		1301
 
+/* SAI */
+#define SOF_TKN_IMX_SAI_MCLK_ID			1400
 #endif
```

Comment:
@groncarolo  

Two comments:

1) Already reserved tokens for SAI, please replace SOF_TKN_IMX_SAI_FIRST_TOKEN with yours.
2) checkpatch.pl warns about indipendently being a typo.

Reply:
Thanks!
I did not see it